rollup.config.js

```diff
@@ -28,14 +28,14 @@ const PLUGINS = [
 module.exports = [
   // browser-friendly UMD build
   {
-    input: 'src/index.ts', // directory to transpilation of typescript
+    input: 'src/browser.ts', // directory to transpilation of typescript
     external: ['cross-fetch', 'cross-fetch/polyfill'],
     output: {
       name: 'window',
       extend: true,
       file: getOutputFileName(
         // will add .min. in filename if in production env
-        resolve(ROOT, pkg.browser),
+        resolve(ROOT, pkg.jsdelivr),
```

Comment:
Why not just `"pkg.umd"`?

Reply:
These are the fields in the `package.json`! Since there is no `umd` field in package.json I added the field I was sure will always contain the `umd` build